foo = ['a', 'b', 'c']
print(foo[3])      # will this result in an error?
# Yes, there isn't anything at [3] to access